### DISCUSSION TITLE
Has the Tesla API performance degraded ?

### DISCUSSION THREAD

**Discussion by hemang249** [timdorr/tesla-api#425] [General]
Idk if this is just on my end, but recently I've seen a huge spike in Tesla API response times.

The /vehicles endpoint, used to respond under 2-3s earlier depending on the number of cars on a single token. Now even with single vehicle, its taking upto 5-7 seconds. Going into 10-15s second territory when the token has > 3 cars on it.

The /vehicle_data endpoint has a similar performance. It used to respond in about 1.5-2s earlier, whereas for the past 1-2 weeks i've seen it take upwards of 5-7 seconds to give back a response.

I am not sure if this is just on my end or everyone else is experiencing the same.

**Comment by BJReplay** [comment]
Very slow response in the app for powerwall status in the last few hours
too.
